pkg/services/sqlstore/alert.go

```diff
@@ -295,6 +296,34 @@ func PauseAlert(cmd *m.PauseAlertCommand) error {
 	})
 }
 
+func PauseDashboardAlerts(cmd *m.PauseDashboardAlertsCommand) error {
+	return inTransaction(func(sess *DBSession) error {
+		if cmd.DashboardId == 0 {
+			return fmt.Errorf("command contains no dashboard id")
+		}
+
+		var buffer bytes.Buffer
```

Comment:
I dont think we need to create a Buffer for this. A more clear way would be to just have one sql statement. 

`sql := "UPDATE alert SET state = ? WHERE dashboard_id = ?`